src/pages/AccountSettings/tabs/OktaAccess/OktaConfigForm/OktaConfigForm.tsx

```diff
@@ -0,0 +1,180 @@
+import { zodResolver } from '@hookform/resolvers/zod'
+import { useState } from 'react'
+import { SubmitHandler, useForm } from 'react-hook-form'
+import { z } from 'zod'
+
+import Banner from 'ui/Banner'
+import BannerContent from 'ui/Banner/BannerContent'
+import Button from 'ui/Button'
+import Icon from 'ui/Icon'
+import TextInput from 'ui/TextInput'
+import Toggle from 'ui/Toggle'
+
+const FormSchema = z.object({
+  clientId: z.string().min(1, 'Client ID is required'),
+  clientSecret: z.string().min(1, 'Client Secret is required'),
+  redirectUri: z.string().url('Redirect URI must be a valid URL'),
+})
+
+type FormValues = z.infer<typeof FormSchema>
+
+export function OktaConfigForm() {
+  const { register, handleSubmit, formState } = useForm<FormValues>({
+    resolver: zodResolver(FormSchema),
+    mode: 'onChange',
+  })
+
+  const [oktaEnabled, setOktaEnabled] = useState(false)
+  const [oktaLoginEnforce, setOktaLoginEnforce] = useState(false)
+  const [showPassword, setShowPassword] = useState(false)
+
+  const onSubmit: SubmitHandler<FormValues> = (data) => {
+    console.log('Form Data: ', data)
+  }
+
+  return (
+    <div className="w-5/6 border-2 border-solid border-ds-gray-primary p-4 text-ds-gray-octonary">
```

Comment:
`m`: Can we move this over to our `Card` component that we have? You should be able to compose/customize it as needed to match the styles given in the designs.

Reply:
You can check the stories for it for guidance or, poke @spalmurray-codecov if you have any deeper questions about it.